### ISSUE TITLE
Add Ability to Specify Categories

### ISSUE THREAD

**Issue by DiamondPrecisionComputing** [pdbme/pdbMate#8]
It would be helpful to be able to specify a category to use in the nzb download client.

**Comment by tblotko** [comment]
Yeah that would be helpful especially when you automate the tasks

**Comment by pdbme** [comment]
You mean one global category set by the user (saved in appsettings.json) that pdbMate sends to the download client when adding nzb files.

Or do you need different categories in pdbMate depending on what?

**Comment by nexalapp** [comment]
@pdbme I am fairly certain the OP is asking about download client categories. I had also looked for this setting when setting up my installation. I do believe one global category in the appsettings.json file would suffice for these requests.
